Xamarin.Forms.Core.Windows.UITests/WindowsTestBase.cs

```diff
@@ -19,6 +19,8 @@ public static IApp ConfigureApp()
 		{
 			if (Session == null)
 				Session = CreateWindowsDriver();
+			else
+				Reset();
```

Comment:
If user gets here but the app is already running then they want it reset to the first page. This is how android/ios work